e2e/README.md

```diff
@@ -135,6 +135,7 @@ We are going to enumerate the variables that will affect the product versions us
 The following environment variables affect how the tests are run in both the CI and a local machine.
 
 - `SKIP_SCENARIOS`: Set this environment variable to `false` if it's needed to include the scenarios annotated as `@skip` in the current test execution. Default value: `true`.
+- `BEATS_LOCAL_PATH`: Set this environment variable to the path to your local copy of Beats if it's needed to use the binary snapshots produced by your local build instead of the official releases. The snapshots will be fetched from the `build/distributions` local directory. Default: empty. This variable is intented to be used by Beats developers, when testing locally the artifacts generated its own build.
 - `BEATS_USE_CI_SNAPSHOTS`: Set this environment variable to `true` if it's needed to use the binary snapshots produced by Beats CI instead of the official releases. The snapshots will be downloaded from a bucket in Google Cloud Storage. Default: `false`. This variable is used by the Beats repository, when testing the artifacts generated by the packaging job.
```

Comment:
```suggestion
- `BEATS_LOCAL_PATH`: Set this environment variable to the path to your local copy of Beats if it's needed to use the binary snapshots produced by your local build instead of the official releases. The snapshots will be fetched from the `build/distributions` local directory. This variable is intented to be used by Beats developers, when testing locally the artifacts generated its own build. Default: empty.
- `BEATS_USE_CI_SNAPSHOTS`: Set this environment variable to `true` if it's needed to use the binary snapshots produced by Beats CI instead of the official releases. The snapshots will be downloaded from a bucket in Google Cloud Storage. This variable is used by the Beats repository, when testing the artifacts generated by the packaging job. Default: `false`.
```